src/main/kotlin/org/elm/lang/core/psi/elements/ElmRecordBaseIdentifier.kt

```diff
@@ -0,0 +1,26 @@
+package org.elm.lang.core.psi.elements
+
+import com.intellij.lang.ASTNode
+import com.intellij.openapi.editor.markup.GutterDraggableObject.flavor
+import com.intellij.psi.PsiElement
+import org.elm.lang.core.psi.ElmPsiElementImpl
+import org.elm.lang.core.psi.ElmTypes.LOWER_CASE_IDENTIFIER
+import org.elm.lang.core.psi.elements.Flavor.*
+import org.elm.lang.core.resolve.ElmReferenceElement
+import org.elm.lang.core.resolve.reference.*
+
+/**
+ * The name of an existing record which is to being updated.
```

Comment:
typo. And I would make it a bit more descriptive. Maybe something like "In a record literal update expression, the name of an existing record which is to be updated."